test/visual/message-list.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+
+<head lang="en">
+  <meta charset="UTF-8" />
+  <title>Message tests</title>
+  <script>
+    window.polymerSkipLoadingFontRoboto = true;
+  </script>
+  <style>
```

Comment:
We could test the scrollability in visual tests by setting e.g. `height: 95px` for the message list.

Reply:
Although maybe we then also need to have another message list without defined height, just to make sure that it is visible without setting the height.

Reply:
I added a new test to the set
```java
it('message list should scroll when height is less than content', () => {
  messageList.style.height = '100px';
  expect(messageList.scrollTop).to.be.equal(0);
  messageList.scrollBy(0, 1000);
  expect(messageList.scrollTop).to.be.at.least(1);
});
```